moving-pictures-tutorial.ipynb

```diff
@@ -6,13 +6,19 @@
    "source": [
     "# \"Moving Pictures\" tutorial\n",
     "\n",
-    "**_note:_ this tutorial is an adaptation of the same tutorial that may be found on the [official QIIME 2 docs website](https://docs.qiime2.org/2019.10/tutorials/moving-pictures/). The original tutorial uses the QIIME 2 CLI interface.**\n",
-    "  \n",
+    "**_note:_ This guide assumes you have nativly installed QIIME 2 using this [procedure](https://docs.qiime2.org/2019.10/install/native/). To execute the script properly, open this notebook in Jupyter Notebook after you activated conda QIIME 2 environment. QIIME 2 does not currently support Windows. We recommend using Ubuntu virtual machines to complete native installation.**\n",
+    "\n",
+    "**_note:_ This tutorial is an adaptation of the same tutorial that may be found on the [official QIIME 2 docs website](https://docs.qiime2.org/2019.10/tutorials/moving-pictures/). The original tutorial uses the QIIME 2 CLI interface.**\n",
+    "\n",
+    "Instead CLI interface, this tutorial uses [Artifact API](https://docs.qiime2.org/2019.10/interfaces/artifact-api/) - a Python 3 application programmer interface (API) for QIIME 2. The Artifact API supports interactive computing with QIIME 2 using the Python 3 programming language. The API is automatically generated, and its availability depends on which QIIME 2 plugins are currently installed. It has been optimized for use in the Jupyter Notebook. The Artifact API is a part of the QIIME 2 framework; no additional software needs to be installed to use it. \n",
+    "\n",
     "**The notebook was tested using the `2019.10` version of QIIME 2.**\n",
     "\n",
-    "In this tutorial you’ll use QIIME 2 to perform an analysis of human microbiome samples from two individuals at four body sites at five timepoints, the first of which immediately followed antibiotic usage. A study based on these samples was originally published in Caporaso et al. (2011). The data used in this tutorial were sequenced on an Illumina HiSeq using the Earth Microbiome Project hypervariable region 4 (V4) 16S rRNA sequencing protocol.\n",
+    "In this tutorial you’ll use QIIME 2 to perform an analysis of human microbiome samples from two individuals at four body sites at five timepoints, the first of which immediately followed antibiotic usage. A study based on these samples was originally published in [Caporaso et al. (2011)](https://www.ncbi.nlm.nih.gov/pubmed/21624126). The data used in this tutorial were sequenced on an Illumina HiSeq using the [Earth Microbiome Project](http://earthmicrobiome.org/) hypervariable region 4 (V4) 16S rRNA sequencing protocol.\n",
     "\n",
-    "## before you start:\n",
+    "> Question 1: Why are 16S gene primers so often used in microbiome studies? What other genes can be used in Next Generation Sequencing in classification of Bacteria and Archaea? What genes do we use to identify other Taxa?  \n",
```

Comment:
Is this a question you came up with? I don't see it in the original CLI tutorial. If yes, I'd try rephrasing it to make it more specific.
For example:
```suggestion
    "> Question 1: Why is 16S rRNA marker gene sequencing so often used to study the microbiome? What other marker genes are commonly used to identify Bacteria and Archaea?  
    What marker genes do we use to identify other microbial kingdoms (e.g. Fungi and microbial Eukaryotes)?  \n",
```